tools.go

```diff
@@ -0,0 +1,12 @@
+// +build tools
+
+package tools
+
+import (
+	_ "github.com/UnnoTed/fileb0x"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [a blank import should be only in a main or test package, or have a comment justifying it](https://app.codacy.com/gh/owncloud/ocis-hello/pullRequest?prid=4957233)